Common/Core/PID/TPCPIDResponse.h

```diff
@@ -0,0 +1,216 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+///
+/// \file   TPCPIDResponse.h
+/// \author Annalena Kalteyer, Christian Sonnabend
+/// \brief
+
+#ifndef O2_PID_TPC_RESPONSE_H_
+#define O2_PID_TPC_RESPONSE_H_
+
```

Comment:
Main comment for this class is that `mBetheBlochParams` needs to be used consistently for all relevant places:
* TPC signal
* Expected signal (BB params and later on ND parametrization)
* Expected sigma
Perhaps also provide a function with consistently returns the correct signal (e.g. `getTPCsignal()`)